semantic-tsx/src/Language/TSX/Tags.hs

```diff
@@ -100,6 +100,19 @@ instance ToTags Tsx.Class where
     } = yieldTag text Class loc byteRange >> gtags t
   tags t = gtags t
 
+instance ToTags Tsx.Module where
+  tags t@Tsx.Module
+    { ann = loc@Loc { byteRange }
+    , name
+    } = match name
+    where
+      match expr = case expr of
+        Prj Tsx.Identifier { text } -> yield text
+        -- TODO: Handle NestedIdentifiers and Strings
+        -- Prj Tsx.NestedIdentifier { extraChildren } -> match
+        _ -> gtags t
+      yield text = yieldTag text Module loc byteRange >> gtags t
```

Comment:
Does this change belong to another PR? (No worries if so, I just want to be sure we don’t merge something somewhere it shouldn’t be prematurely/by mistake.)